src/routing/admin/opportunity/opportunity.tsx

```diff
@@ -68,7 +70,8 @@ export const OpportunityRoutes: FC<Props> = ({ paths }) => {
   const community = data?.ecoverse?.opportunity?.community;
   const parentMembers = challengeData?.ecoverse?.challenge.community?.members || [];
   const opportunityUUID = data?.ecoverse.opportunity.id || '';
-  if (loadingOpportunity || loadingChallenge) return <Loading text={'Loading'} />;
+  if (loadingOpportunity) return <Loading text={'Loading opportunity'} />;
+  if (loadingChallenge) return <Loading text={'Loading challenge'} />;
```

Comment:
"Loading challenge", when you are in opportunity context seems odd.
Also, it needs translation.

Reply:
Yep I agree it is strange but it will be handled in #1174 
With the routing optimizations all of this will be moved somewhere else and  it can disappear.